src/p_misc.c

```diff
@@ -992,76 +1732,130 @@ prim_ext_name_okp(PRIM_PROTOTYPE)
     oper1 = POP();
 
     if (oper1->type != PROG_STRING)
-	abort_interp("Object name string expected (1).");
+        abort_interp("Object name string expected (1).");
 
     char *b = DoNullInd(oper1->data.string);
 
     if (oper2->type == PROG_STRING) {
-	strcpyn(buf, sizeof(buf), DoNullInd(oper2->data.string));
-	for (ref = 0; buf[ref]; ref++)
-	    buf[ref] = tolower(buf[ref]);
-	if (!strcmp(buf, "e") || !strcmp(buf, "exit")) {
+        strcpyn(buf, sizeof(buf), DoNullInd(oper2->data.string));
+
+        for (ref = 0; buf[ref]; ref++)
+            buf[ref] = tolower(buf[ref]);
+
+        /**
+         * @TODO Maybe these should use case-insenstive?
```

Comment:
Agreed.  I'd take it a step further and strncmp them based on the length of the input string.

So for example, "ex" would match "exit".  But doing case insensitive is trivial so we should definitely do that :D

Reply:
Might need a special case for "p" - does it match "player" or "program"? Also, should "m" and "mu" also be aliases for "muf"?

I'm sure it's doable.